service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -119,6 +131,34 @@ public void unadoptSite(JWTData userData, int siteId) {
         .execute();
   }
 
+  @Override
+  public void forceUnadoptSite(JWTData userData, int siteId) {
+    isAdminCheck(userData.getPrivilegeLevel());
+    checkSiteExists(siteId);
+    if (!isAlreadyAdopted(siteId)) {
+      throw new WrongAdoptionStatusException(false); // wrong error?
+    }
+
+    AdoptedSitesRecord adoptedSite = db.selectFrom(ADOPTED_SITES)
+            .where(ADOPTED_SITES.SITE_ID.eq(siteId))
+            .fetchInto(AdoptedSitesRecord.class)
+            .get(0);
+
+    Integer adopterId = adoptedSite.getUserId();
+
+    UsersRecord adopter = db.selectFrom(USERS)
+            .where(USERS.ID.eq(adopterId))
+            .fetchOne();
+
+    if(isAdmin(adopter.getPrivilegeLevel()) && !(userData.getPrivilegeLevel() == (PrivilegeLevel.SUPER_ADMIN))) {
```

Comment:
⛏️ Super small – this is using `==` for enum equality, but above it uses `.equals`. Both are correct, but we should keep it consistent